### ISSUE TITLE
Tarball for xkbcommon-1.1.0 missing

### ISSUE THREAD

**Issue by primeos** [xkbcommon/website#2]
xkbcommon-1.1.0 was tagged 2 days ago (https://github.com/xkbcommon/libxkbcommon/releases/tag/xkbcommon-1.1.0) but there's no entry/tarball on the website yet (https://xkbcommon.org/download/libxkbcommon-1.1.0.tar.xz) and no announcement on `wayland-devel` (https://lists.freedesktop.org/archives/wayland-devel/2021-March/thread.html).

Is this intentional or was it maybe forgotten?
And would you recommend Linux distributions to fetch the tarball via your website or GitHub?:
- https://github.com/xkbcommon/libxkbcommon/archive/xkbcommon-1.1.0.tar.gz (already available)
- https://xkbcommon.org/download/libxkbcommon-1.1.0.tar.xz (not yet available)

From a quick comparison the only difference seems to be that GH's tarball contains three additional files (apart from the different compression algorithms and top-level directory name):
```diff
+.github
+.github/workflows
+.github/workflows/main.yml
```

---

And if you don't mind: It might be nice to add a link to the website on https://github.com/xkbcommon/libxkbcommon as well (in GH's "Edit repository details" UI one can add a "Description" and "Website").

Anyway, thanks for maintaining xkbcommon! :)

**Comment by bluetech** [comment]
Yes, sorry about that; I temporarily don't have access to the xkbcommon.org website (which hosts the tarballs and documentation), but I had only found that out after pushing the tag. I hope to sort it out ASAP. If not, I will find another solution.

> From a quick comparison the only difference seems to be that GH's tarball contains three additional files (

The tarballs are generated with `meson dist`, while github does a `git archive`, I guess these are the differences.

> And if you don't mind: It might be nice to add a link to the website

Added, thanks for the suggestion.
